src/main/java/com/adobe/aem/compgenerator/utils/DialogUtils.java

```diff
@@ -152,7 +203,8 @@ private static Element createPropertyNode(Document document, Node currentNode, P
                     actualField.setAttribute(Constants.PROPERTY_CQ_MSM_LOCKABLE, "./" + property.getField());
 
                     Property prop = property.getItems().get(0);
-                    actualField.setAttribute(Constants.PROPERTY_SLING_RESOURCETYPE, getSlingResourceType(prop.getType()));
+                    actualField.setAttribute(Constants.PROPERTY_SLING_RESOURCETYPE,
```

Comment:
For some of these code wraps that are not any different code than before, please undo if you can - the code was already very legible and wrapping at 80 or even 100 char is not necessary in the modern day of monitors that are large enough to easily view 120+ character lines.

It's another nitpick, but just trying to keep the codebase consistent and free of arbitrary changes back and forth between developers with different IDE settings.

Reply:
Updated